stats/opencensus/e2e_test.go

```diff
@@ -237,6 +237,23 @@ func distributionDataLatencyCount(vi *viewInformation, countWant int64, wantTags
 	return nil
 }
 
+// waitForServerCompletedRPCs waits until both Unary and Streaming metric rows
+// appear for server completed RPC's view (by checking for length of rows to be
+// 2). Returns an error if both the Unary and Streaming metric not found within
+// the passed context's timeout.
+func waitForServerCompletedRPCs(ctx context.Context, fe *fakeExporter) error {
+	for ; ctx.Err() == nil; <-time.After(time.Millisecond) {
+		rows, err := view.RetrieveData("grpc.io/server/completed_rpcs")
```

Comment:
Since `err` from `RetrieveData` is ignored here, why not nix the variable completely? and remove the following non-nil `err` check

Reply:
err is not ignored. err determines whether to continue through the for.